src/build.jl

```diff
@@ -18,41 +18,78 @@ end
 
 Base.summary(build::BuildRef) = string(build.repo, SHA_SEPARATOR, snipsha(build.sha))
 
+# reduce download bandwidth and time by keeping copies after the build
+# TODO: intercept JLDOWNLOAD instead?
+function sync_srcs!(fromdir, todir, link::Bool)
+    mkpath(fromdir)
+    mkpath(todir)
+    for n in readdir(fromdir)
+        endswith(n, ".tmp") && continue
+        src = abspath(fromdir, n)
+        dst = abspath(todir, n)
+        if isfile(src) && !islink(src) && !ispath(dst)
+            if link
+                symlink(src, dst)
+            else
+                dsttmp = dst * ".tmp"
+                cp(src, dsttmp, force=true)
+                chmod(dsttmp, 0o444)
+                Base.Filesystem.rename(dsttmp, dst, force=true) # mv is implemented badly in Base, so avoid it
```

Comment:
Can you open an upstream issue for this if there isn't one? (And perhaps link one if there is?)

Reply:
I assume someone did it intentionally (because of Windows), it just is bad here